### ISSUE TITLE
Migrate from Docusaurus 1 to Docusaurus 3

### ISSUE THREAD

**Issue by HonkingGoose** [prettier/prettier#11409]
I noticed you're still using Docusaurus 1:

https://github.com/prettier/prettier/blob/493f393ac7fd4b072a00d6600587308cde32b010/website/package.json#L21

The developers of Docusaurus 2 are confident that it's ready for mainstream adoption now, as they state on their blog post here: https://docusaurus.io/blog/2021/05/12/announcing-docusaurus-two-beta

Read the migration guide from Docusaurus 1 to Docusaurus 2 to learn more: https://docusaurus.io/docs/migration
Docusaurus has a CLI tool that can do some migration steps automatically, but there are still manual steps to do as well.

**Comment by sosukesuzuki** [comment]
I didn't know about docusaurus v2, thank you for share this information! We will update docusaurus when version 2 is officially released.

**Comment by fisker** [comment]
I've followed the progress for a long time, maybe we can open a PR to prepare for it. But I agree with @sosukesuzuki, we should wait for official release to merge.

**Comment by HonkingGoose** [comment]
> I didn't know about docusaurus v2, thank you for share this information!

You're welcome. 😉 

> But I agree with @sosukesuzuki, we should wait for official release to merge.

Sounds like a good plan!

**Comment by Josh-Cena** [comment]
Yep, if I've got time I would volunteer to take a look at this (no plans yet)

**Comment by fisker** [comment]
@Josh-Cena Any progress?

**Comment by Josh-Cena** [comment]
Sorry! I have very limited time these weeks. I have made some basic progress, but I'm stuck in the same place as Babel's website: the playground is very tricky to integrate into the React SPA... Sorry if I'm blocking someone else to work on this. I will I assign myself but I'll continue to investigate when I have time.

**Comment by fisker** [comment]
No rush, just checking, take you time.

**Comment by sosukesuzuki** [comment]
I'll working on this.

**Comment by tats-u** [comment]
v3.0.0-alpha.0 is now avaiable.
https://github.com/facebook/docusaurus/releases/tag/v3.0.0-alpha.0

In v3, MDX v2 instead of v1 must be used.

**Comment by tats-u** [comment]
https://docusaurus.io/blog/preparing-your-site-for-docusaurus-v3

v3 seems to be released soon.
beta.0 is now available.
